sim-lib/src/lib.rs

```diff
@@ -139,6 +139,12 @@ pub struct ActivityParser {
     /// The destination of the payment.
     #[serde(with = "serializers::serde_node_id")]
     pub destination: NodeId,
+    /// The time in the simulation to start the payment
+    #[serde(default)]
+    pub start_secs: u16,
```

Comment:
`#[serde(rename(deserialize = "start"))]` to deserialize from activity field name `start`

Reply:
@enigbe I decided to switch from `start` to `start_secs` to be consistent with `interval_secs` and to make sure the user knows what units to use.